AllStatusMessagesForTsDeployment.xml

```diff
@@ -0,0 +1,22 @@
+<ActionDescription Class="Executable" DisplayName="ShowStatusMessages" MnemonicDisplayName="MnemonicShowStatusMessages" 

```

Comment:
I'd prefer not to put this in the root of the repository. How about src/Console/Extensions/Actions/ ?

Reply:
How does it work? Can I change location in this pull request or need somehow to abandon it and try again?

Reply:
clone your branch on your local machine (vscode makes this very simple)
use git mv "path to source file" "path to destination file"
git commit -m "moving file"
git push